modules/osqp-cpp/20231004.0.bcr.1/presubmit.yml

```
@@ -0,0 +1,18 @@
+matrix:
+  platform:
+  - debian10
+  - ubuntu2004
+  - macos
+  - macos_arm64
+  - windows
+  bazel:
+  - 7.x
+  - 6.x
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_targets:
+    - '@osqp-cpp//:osqp++'
+    - '@osqp-cpp//:license'
```

Consider the wildcard to keep it easier to maintain in the future.

```suggestion
    - '@osqp-cpp//...'
```